Tests/NIOTransportServicesTests/NIOTSBootstrapTests.swift

```diff
@@ -246,6 +246,72 @@ final class NIOTSBootstrapTests: XCTestCase {
         XCTAssertNil(NIOTSListenerBootstrap(validatingGroup: wrongELG, childGroup: correctELG))
         XCTAssertNil(NIOTSListenerBootstrap(validatingGroup: wrongEL, childGroup: correctEL))
     }
+    
+    func testEndpointReuseShortcutOption() throws {
+        let group = NIOTSEventLoopGroup()
+        let listenerChannel = try NIOTSListenerBootstrap(group: group)
+            .bind(host: "127.0.0.1", port: 0)
+            .wait()
+        
+        var optionValue : EventLoopFuture<Bool>? = nil
+        let bootstrap = NIOClientTCPBootstrap(NIOTSConnectionBootstrap(group: group),
+                                              tls: NIOInsecureNoTLS())
+            .channelConvenienceOptions([.allowLocalEndpointReuse])
+            .channelInitializer { channel in
+                optionValue = channel.getOption(NIOTSChannelOptions.allowLocalEndpointReuse)
```

Comment:
Can't we just get the option from `client` once we've waited on the `connect` below? (and elsewhere in this PR)

Reply:
Yes, you're right - done.